2019/Matplotlib_ideas_list.md

```diff
@@ -1 +1,31 @@
+__Name:__ matplotlib.org
+__Description:__ Python 2D plotting library which produces publication quality figures in a variety of hardcopy formats and interactive environments across platforms.
+__website:__ https://matplotlib.org/
+__repository:__ https://github.com/matplotlib/matplotlib
+__email:__ matplotlib@numfocus.org
 
+
+__Project name:__ Matplotlib for Data Scientists
+
+__Description:__ Matplotlib was primarily develped by scientists for scientific visualization, and modeled on scientific computation languages such as Matlab, but one of the fastest growing audiences for Matplotlib is data science practitioners. To address the needs of this audience, we propose developing a conceptual overview (such as the scikit one linked below) oriented at newcomers to scientific visualization. 
+
+The overview would possibly cover topics such as:
+* the visualization pipeline: data->transformation->chart
+* matplotlib's role in the viz pipeline , i.e. that it's a low level library
+* assumptions matplotlib makes about the structure of the data, user, and task
+* the relationship between matplotlib and libaries built on top, such as Pandas and Seaborn
```

Comment:
can you give information on this topic? Preferably a blog post/ Stack Overflow posts that already exist and aren't very outdated.